provisioning/setup.yaml

```diff
@@ -1,22 +1,22 @@
 hare:
   post_install:
-    cmd: /opt/seagate/cortx/hare/bin/hare_setup
-    args: --report-unavailable-features --configure-logrotate --post_install
+    cmd: /opt/seagate/cortx/hare/bin/hare_setup post_install
+    args: --report-unavailable-features --configure-logrotate
   init:
-    cmd: /opt/seagate/cortx/hare/bin/hare_setup
-    args: --init
+    cmd: /opt/seagate/cortx/hare/bin/hare_setup init
+    args:
   config:
-    cmd: /opt/seagate/cortx/hare/bin/hare_setup
-    args: --config $URL --filename '/var/lib/hare/cluster.yaml'
+    cmd: /opt/seagate/cortx/hare/bin/hare_setup config
+    args: --config $URL --filepath '/var/lib/hare/cluster.yaml'
   test:
-    cmd: /opt/seagate/cortx/hare/bin/hare_setup
-    args: --test
+    cmd: /opt/seagate/cortx/hare/bin/hare_setup test
```

Comment:
Are you sure that `test` should go to `cmd`, not to `args`?

Reply:
@knekrasov  I think it should go to cmd. Please see below from Mini-Provisioner spec doc
The setup.yaml specified by any component exhibits below format: 

```
<component>:   

  post_install:  				# Example: motr_setup post_install --config $URL 

    cmd: <component>_setup post_install	# CI setup command implemented by Component 

    args: --config <URL>			# ConfStore Config URL provided by Provisioner 
```